### DISCUSSION TITLE
How to give the path of local image to add in canvas.

### DISCUSSION THREAD

**Discussion by PatilRupaliAress** [iddan/react-native-canvas#245] [Q&A]
I was loading the local images which are stored in project. But this images are not loading in canvas.But when I load the images from web its working fine. What should I do ?

**Comment by Nours312** [comment]
@PatilRupaliAress, 
for Android : you need to store your image on /android/app/src/asset/whatYouWant/img.png
for IOS you need to create a symlink with Xcode on /whatYouWant/img.png

And use it with like this : 
```js
const file : 'whatYouWant/img.png' ;
const image = new Image(canvas);
image.addEventListener('load', () => {
            ctx.drawImage(image, x, y, w, h);
});
image.src = (Platform.OS === 'android' ? 'file:///android_asset/' : '') + file ;
```

**Comment by ShaoGongBra** [comment]
Does this only work in the release version? How to use the debug version?
